Backend/src/main/java/ca/mcgill/cooperator/controller/ControllerUtils.java

```diff
@@ -560,6 +560,20 @@ public static CourseOfferingDto convertToDto(CourseOffering co) {
         }
         return courseOfferingDtos;
     }
+    
+    public static Set<CourseOfferingDto> convertCourseOfferingSetToDto(
+            Set<CourseOffering> courseOfferings) {
```

Comment:
Change `Set` to `Collection` so that it's more general and works for both sets and lists